### PR TITLE
NEPT-1151: Implement classes in headings.

### PR DESCRIPTION
**Pull request by Fefaine**

### CODE REVIEW

**Comment by gillesdeudon** [comment]
👍

**Comment by voidtek** [comment]
:+1:

File: templates/html_tag/html_tag.component.inc [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+/**
+ * @file
+ * Contains component file.
+ */
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function europa_preprocess_html_tag(&$variables, $hook) {
+  $headers = ['h1', 'h2', 'h3', 'h4', 'h5'];
```
Comment (drupol) [inline templates/html_tag/html_tag.component.inc]: Shouldn't we use the old array convention ?

I don't care to use the new one or the old one, it's just that mixing both is crap.
Drupal 7 code should be using old array style, even if we are using PHP >= 5.4.

How can we fix this once for all ?

@gillesdeudon @voidtek @imanoleguskiza @Fefaine

Comment (haringsrob) [inline templates/html_tag/html_tag.component.inc]: Use phpcs rules on pre push and CI

Comment (drupol) [inline templates/html_tag/html_tag.component.inc]: @haringsrob We are using GrumPHP in Atomium but we should also use it here indeed.

Comment (drupol) [inline templates/html_tag/html_tag.component.inc]: @haringsrob We have grumPHP enabled on the **europa-atomium** branch but if the colleagues doesn't do "composer install" in their repository, GrumPHP is not enabled.

**Pull request merged**